README.md

```diff
@@ -4,9 +4,7 @@
 
```

Comment:
Actually, looks like the docs shield is still linking to 1.x. Can you change it to `https://media.twiliocdn.com/sdk/ios/video/releases/2.0.0-preview6/docs`?